syntax = "proto3";

package person;

message PersonRequest {
  // The first name
  string firstName = 1;

  // The last name
  string lastName = 2;

  // The email
  string email = 3;

  // Is the person an employee?
  bool isEmployee = 4;
}

message PersonResponse {
  // Ack
  bool ok = 1;
}

service PersonSender {
  rpc SendPerson (PersonRequest) returns (PersonResponse) {}
}
